src/Android/Accessibility/AccessibilityHelpers.cs

```diff
@@ -390,12 +390,15 @@ private static string ExtractUri(string uri, AccessibilityNodeInfo addressNode,
                 var hasHttpProtocol = uri.StartsWith("http://") || uri.StartsWith("https://");
                 if (!hasHttpProtocol && uri.Contains("."))
                 {
-                    if (Uri.TryCreate("http://" + uri, UriKind.Absolute, out var uri2))
+                    if (Uri.TryCreate("https://" + uri, UriKind.Absolute, out var _))
+                    {
+                        return string.Concat("https://", uri);
+                    } else if (Uri.TryCreate("http://" + uri, UriKind.Absolute, out var _))
```

Comment:
Please move the `else if` on the next line after the trailing `}`

Reply:
Also, this `else if` block is not necessary, so you should be able to remove it anyway. If `https://` doesn't work, `http://` won't either when using `Uri.TryCreate()`